src/textual/widgets/_header.py

```diff
@@ -161,11 +161,19 @@ def _on_click(self):
         self.toggle_class("-tall")
 
     def _on_mount(self, _: Mount) -> None:
-        def set_title(title: str) -> None:
+        def set_title() -> None:
+            screen_title = self.screen.title
```

Comment:
We should have a more elegant way of exposing the actual title. So that a developer doesn't need to replicate this logic.

Perhaps a computed value called `screen_title` and `screen_subtitle`

Reply:
See 4ed93d45c